pkg/kubelet/disk_manager.go

```diff
@@ -31,11 +31,14 @@ import (
 // mb is used to easily convert an int to an mb
 const mb = 1024 * 1024
 
+// ImageFsInfoFunc is the interface for getting ImageFsInfo.
+type ImageFsInfoFunc func() (cadvisorapi.FsInfo, error)
+
 // Implementation is thread-safe.
 type diskSpaceManager interface {
 	// Checks the available disk space
 	IsRootDiskSpaceAvailable() (bool, error)
-	IsRuntimeDiskSpaceAvailable() (bool, error)
+	IsRuntimeDiskSpaceAvailable(ImageFsInfoFunc) (bool, error)
```

Comment:
I think the `ImageFsInfoFunc` should be an implementation detail, not part of the interface.